cmd/olfullnode/show_node_id.go

```diff
@@ -64,6 +66,12 @@ func showNodeID(root string, shouldShowIP bool) (result string, err error) {
 		}
 		return fmt.Sprintf("%s@%s", nodeKey.ID(), u.Host), nil
 	} else {
-		return string(nodeKey.ID()), nil
+		pubKey, _ := keys.PubKeyFromTendermint(nodeKey.PubKey().Bytes())
```

Comment:
why this logic only exists in the scenario that we don't show IP? is this intentional?